docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -197,7 +202,7 @@
       block:
       - name: Find deployment-manager app helm-chart in application directory
         find:
-          paths: /usr/local/share/applications/helm/
+          paths: application_helm_chart_dir
           patterns: 'deployment-manager*.tgz'
         register: app_helm_chart
```

Comment:
What's the app_helm_chart usage here if it was uploaded as manager_app_chart in line215?